<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicalIntelligenceClass</class>
 <widget class="QMainWindow" name="GraphicalIntelligenceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphicalIntelligence</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lblJuiz">
          <property name="text">
           <string>Juiz:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtJuiz"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblTempoJogo">
          <property name="text">
           <string>Tempo restante:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtTempoJogo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lblPlacar">
          <property name="text">
           <string>Placar</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="txtPlacarAmarelo"/>
          </item>
          <item>
           <widget class="QLabel" name="lblX">
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPlacarAzul"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblNrTimeouts">
          <property name="text">
           <string># Timeouts:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtNrTimeoutsAzul"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblTempoTimeouts">
          <property name="text">
           <string>Tempo Timeouts:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtTempoTimeoutsAzul"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="txtNrTimeoutsAmarelo"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="txtTempoTimeoutsAmarelo"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="chkUpdate">
          <property name="text">
           <string>Update</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkForceInt">
          <property name="text">
           <string>For√ßar inteligencia</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Stage size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtStageWidth"/>
        </item>
        <item>
         <widget class="QLineEdit" name="txtStageLenght"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="StageView" name="stageView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StageView</class>
   <extends>QGraphicsView</extends>
   <header>stageview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GraphicalIntelligence.qrc"/>
 </resources>
 <connections/>
</ui>
